appcenter-data/Data.d.ts

```diff
@@ -2,7 +2,6 @@
 // Licensed under the MIT License.
 
 export const enum TimeToLive {
-    INFINITE = -1,
```

Comment:
Why is this removed?

Reply:
The default is now infinite so was doing some cleanup but I'll put it back because its still used somewhere.